specs/electra/beacon-chain.md

```diff
@@ -1194,21 +1282,25 @@ def apply_deposit(state: BeaconState,
     if pubkey not in validator_pubkeys:
         # Verify the deposit signature (proof of possession) which is not checked by the deposit contract
         if is_valid_deposit_signature(pubkey, withdrawal_credentials, amount, signature):
-            add_validator_to_registry(state, pubkey, withdrawal_credentials, amount)
+            add_validator_to_registry(state, pubkey, withdrawal_credentials, Gwei(0))  # [Modified in Electra:EIP7251]
```

Comment:
what about
```
if pubkey not in validator_pubkeys:
  if I!s_valid_deposit_signature(pubkey, withdrawal_credentials, amount, signature):
    break
 add_validator_to_registry(state, pubkey, withdrawal_credentials, Gwei(0))
state.pending_deposits.append(PendingDeposit(
                pubkey=pubkey,
                withdrawal_credentials=withdrawal_credentials,
                amount=amount,
                signature=signature,
                slot=GENESIS_SLOT,
            ))
```

Reply:
oops missed this comment, yeah just readability, as the appending logic is duplicated as far as I could tell

Reply:
I think the status quo is more readable.